src/test/java/study/StringTest.java

```diff
@@ -0,0 +1,52 @@
+package study;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+public class StringTest {
+
+    @Test
+    void 요구사항1_split시_1과2분리_확인() {
+        String givenString = "1,2";
+
+        String[] split = givenString.split(",");
```

Comment:
```suggestion
        final String[] actual = "1,2".split(",");
```

로컬 변수 없이 구현해도 되지 않을까?

Reply:
BDD 양식에 맞추어서
given when then 을 맞춰서 연습 중 이여서 그렇게 해봤습니다 :)
가시성이 후자가 더 좋다면 후자로 테스트 생성하겠습니다.